src/utils/contracts.ts

```diff
@@ -35,6 +35,14 @@ export const FAUCET_ADDRESSES: {[key in Environment]: string} = {
   [Environment.mainnet]: '0x0000000000000000000000000000000000000000',
 } as const
 
+export const CXIP_ERC721_ADDRESSES: {[key in Environment]: string} = {
+  [Environment.localhost]: '0x0000000000000000000000000000000000000000',
+  [Environment.experimental]: '0x0000000000000000000000000000000000000000',
+  [Environment.develop]: '0x156C8b069232986c8C6bD9017BBAA098e97f0269',
+  [Environment.testnet]: '0x0000000000000000000000000000000000000000',
```

Comment:
Should we set an address for testnet and mainnet so we dont have to backtrack?

Reply:
yeah good call